azure/const.go

```diff
@@ -29,7 +29,7 @@ const (
 	// for annotation formatting rules.
 	RGTagsLastAppliedAnnotation = "sigs.k8s.io/cluster-api-provider-azure-last-applied-tags-rg"
 
-	// ReplicasManagedByAutoscalerAnnotation is set to true in the corresponding capi machine pool
-	// when an external autoscaler manages the node count of the associated machine pool.
+	// ReplicasManagedByAutoscalerAnnotation is deprecated, replaced by the CAPI annotatation
+	// "cluster.x-k8s.io/replicas-managed-by".
```

Comment:
since this const is no longer used anywhere, should we just remove it?

Reply:
fair